assets/stylesheets/pages/_landing.scss

```
@@ -6,6 +6,15 @@
   strong {
     font-weight: 600;
   }
+
+  span {
+    display: block;
+    padding-top: .5rem;
+  }
+
+  @media only screen and(max-width: 767px) { // same media query as in _mobile.css
```

```suggestion
  @media only screen and(max-width: 767px) {
```

While working on the top navigation it became clear to me that we need to refactor these into Sass variables. I can open a PR for this once both this and the top navigation PR are merged.